app/views/includes/head.html

```diff
@@ -1,3 +1,4 @@
 {% include "includes/scripts_ga_1.html" %}
 <link href="/public/stylesheets/application.css" media="screen" rel="stylesheet" type="text/css" />
 <!--[if IE 8]><link rel="stylesheet" media="screen" href="/public/stylesheets/application-ie8.css"/><![endif]-->
+<title>{% block title %}{% endblock %} Probate</title>
```

Comment:
Just curious - if this PR is to add tests, how come we've needed to change html in way that affects end user?

Reply:
That's a bug according to Pa11y so as per the HMCTS Accessibility guidelines  i fixed that issue and added that test to Accessibility pack.